### ISSUE TITLE
Maven cannot download the dependency of ShardingSphere 5.3.1

### ISSUE THREAD

**Issue by KouShenhai** [apache/shardingsphere#24126]
### Which version of ShardingSphere did you use?
My version is ShardingSphere 5.3.1

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
I use ShardingSphere-JDBC

### Operation steps
I imported ShardingSphere in the SpringBoot project and reported an error.
```
            <dependency>
                <groupId>org.apache.shardingsphere</groupId>
                <artifactId>shardingsphere</artifactId>
                <version>5.3.1</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
 ```
``` 
Could not transfer artifact org.springframework:spring-jdbc:jar:${springframework.version} 
```
After repeated debugging, the original pom.xml of the parent module does not define the <springframework.version/> variable.

I hope to solve this problem. Thank you very much.

**Comment by linghengqian** [comment]
- Duplicated with https://github.com/apache/shardingsphere/pull/23355, this issue can be closed and will be resolved in 5.3.2.

- By the way, I don't think it is routine to import ShardingSphere's pom.

**Comment by KouShenhai** [comment]
> * Duplicated with [Remove version management for spring-jdbc #23355](https://github.com/apache/shardingsphere/pull/23355), this issue can be closed and will be resolved in 5.3.2.
> * By the way, I don't think it is routine to import ShardingSphere's pom.

Yeah, I just want to import the ShardingSphere - JDBC dependency, but I can't import it. Therefore, directly import the entire ShardingSphere dependency

Finally, thank you very much!

**Comment by TeslaCN** [comment]
Thanks for your answer! @linghengqian
